sqlmesh/core/engine_adapter/databricks.py

```diff
@@ -137,7 +137,13 @@ def fetchdf(
         return df
 
     def set_current_catalog(self, catalog_name: str) -> None:
+        # Since Databricks splits commands across the Dataframe API and the SQL Connector
+        # (depending if databricks-connect is installed and a Dataframe is used) we need to ensure both
+        # are set to the same catalog since they maintain their default catalog seperately
```

Comment:
@izeigerman note this nuance for Databricks when making future Spark related changes